use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,217 @@
+  * [Part 1 – The Host](#part-1--the-host)
+  * [Part 2 – The Container](#part-2--the-container)
+* [Test Your Setup](#Test-Your-Setup)
+
+![](./images/NIC%20passthrough%20Diagram.png)
+
+
+In this guide we walk through the process of passing a physical NIC into Kata Containers. The "usual" way that a Kata Container is wired for communication is through a bridge and virtual NIC, as can be seen in the image above (marked as default).
+
+For some use cases, the container demands a direct link to the physical port of the host device, for example in a situation were the container is required to route high BW traffic without having support for acceleration such as SR-IOV.
+
+## Before you start (Restrictions, Requirements and Assumptions)  
+
+The method described in this guide relies on your system [supporting  IOMMU.]( https://en.wikipedia.org/wiki/Input–output_memory_management_unit#Published_specifications) Before you start, make sure that your system has this attribute. If it does not, you should look for a different solution then the one presented here. 
+
+If you are uncertain about your system’s support, the first two steps in [Part 1 – The Host](#part-1--the-host) should help you figure this out. 
+
+ This guide was tested using *Intel(R) Xeon(R) CPU D-1548 @ 2.00GHz* and [support for VT-d](https://en.wikipedia.org/wiki/X86_virtualization#Intel-VT-d), Operated by Ubuntu 18.04.3 LTS. The general steps proposed is this guide should match other hardware and OS with some modifications.
+
+This guide’s starting point is a system with an installed and working Docker and Kata Containers.
+
+If that is not the case for you:
+
+- [How to install Kata Containers](https://github.com/kata-containers/documentation/tree/master/install)
+
+- [How to install Docker](https://github.com/kata-containers/documentation/blob/master/install/docker/ubuntu-docker-install.md) 
+
+
+
+## Pass a physical NIC into a Kata Container 
+
+### Part 1 – The Host
+
+**Note:** All next commands are run as a super-user. Use `$ sudo -i` to become a super-user.
+
+
+
+#### Configure system to load with IOMMU support.
+
+   Edit grub file ( `/etc/default/grub` )
+
+```bash
+$ sudo vi /etc/default/grub
+```
+
+And add ` intel_iommu=on iommu=pt` To the **GRUB_CMDLINE_LINUX** line 
+
+> GRUB_CMDLINE_LINUX=" … quiet intel_iommu=on iommu=pt"
+
+
+
+If using UEFI: (like I do) 
+
+```bash
+$ sudo grub2-mkconfig -o /boot/efi/EFI/ubuntu/grub.cfg
+```
+
+Else (not using UEFI):
+
+```bash
+$ sudo grub2-mkconfig -o /boot/grub2/grub.cfg
+```
+
+#### Test that the system loads with IOMMU:
+
+```bash
+$ sudo dmesg | grep -ie iommu
+```
+
+
+#### Enable the *hotplug_vfio_on_root_bus* configuration in the `kata configuration.toml` file:
+
+```bash
+$ sudo vi /usr/share/defaults/kata-containers/configuration.toml
+```
+a. Set hotplug_vfio_on_root_bus = true
+
+b. Make sure you are using the PC machine type by verifying machine_type = "pc"
+
+
+
+#### Install *lshw* and *pciutils* (Ubuntu)
```

Comment:
I would consider making the title more generic (easier to read and also won't need to be updated if we later find we need to add more tools)

> #### Install required tools for Ubuntu